modules/@ergo/core/components/Grid/Grid.vue

```diff
@@ -53,6 +53,14 @@ export default {
             type: Array,
             default: () => [],
         },
+        isAdvancedFilters: {
```

Comment:
Like I said it could be better to not add an additional flag to the Header about filters. We could handle advanced filters from Grid not from Header - we would refactor a lot of code.

As you already added slot to header for new items, we could build that structure from Grid.